server/odc-core/src/main/java/com/oceanbase/odc/core/alarm/AlarmEvent.java

```diff
@@ -16,18 +16,20 @@
 
 package com.oceanbase.odc.core.alarm;
 
+import java.util.Map;
+
 import lombok.Getter;
 
 @Getter
 public class AlarmEvent {
 
     private final String eventName;
 
-    private final String eventMessage;
+    private final Map<String, Object> eventMessage;
```

Comment:
object is a very generalized implement for java. It's there any common  feature for what you want to collect

Reply:
oo，all string